src/lib/compiler/compileData.js

```diff
@@ -817,7 +816,7 @@ export const precompileMusic = (scenes, music) => {
     .map((track, index) => {
       return {
         ...track,
-        dataName: `music_${uuid().replace(/-.*/, "")}${index}`
+        dataName: (`music_track_`+ (index + 101) + '_')//`music_${uuid().replace(/-.*/, "")}${index}`
```

Comment:
remove the comment

Reply:
Cool, also realized i don't need uuid imported anymore.